test/inductor/test_flex_attention.py

```diff
@@ -438,6 +438,47 @@ def test_builtin_score_mods_different_seqlen(
             S,
             D,
         )
+    
+        
+    @supported_platform
+    @common_utils.parametrize("dtype", test_dtypes)
+    @common_utils.parametrize("score_mod", [_identity, _causal])
+    def test_strided_inputs(self, dtype: torch.dtype, score_mod: Callable):
```

Comment:
Could you add some more tests with other permutations of the input matrices?

Reply:
Added more strided inputs https://github.com/pytorch/pytorch/pull/128915/commits/947a1e0237bf0d089ec6f716546139d4018d023c